src/ansys/platform/instancemanagement/definition.py

```diff
@@ -15,7 +15,7 @@
 
 @dataclass(frozen=True)
 class Definition:
-    """Definition of a product that can be started using the product instance management API.
+    """Provides a definition of a product that can be started using the PIM API.
```

Comment:
Usually the docstring includes the "Parameters" section. However, you have all separate docstrings and I didn't want to make changes without asking why.

Reply:
Is this section even used for class documentation?

Right now the generated doc puts the attributes like this, I think this is fine:

![image](https://user-images.githubusercontent.com/53424602/164408352-6773b6ee-632e-494e-9aa4-f22c2846a6ea.png)

Reply:
@plule-ansys I was trying to remember if classes had parameters before asking this--but didn't take the time to look it up. Thanks!